device/sycl/src/seeding/seed_finding.sycl

```diff
@@ -55,6 +60,40 @@ class select_seeds;
 
 }  // namespace kernels
 
+template <class kernelName>
+vecmem::data::vector_buffer<device::prefix_sum_element_t> make_prefix_sum_buff(
+    const std::vector<device::prefix_sum_size_t> sizes,
+    const std::unique_ptr<vecmem::copy>& copy,
+    const traccc::memory_resource& mr, queue_wrapper& queue) {
```

Comment:
The `traccc::sycl::queue_wrapper` type is useful for passing around `::sycl::queue` objects in "pure C\+\+" code. Since here we are defining this function in a source file that has to be built by a SYCL compiler, you could switch to just pass a `::sycl::queue&` non-const reference to the function directly as well.